qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -158,7 +154,7 @@ public void toXmlFastqTest() throws ParserConfigurationException {
 	}
 
 	@Test
-	public void toXmlTest() throws IOException, DOMException, ParserConfigurationException {		
+	public void toXmlTest() throws Exception {		
```

Comment:
are any exceptions thrown from this method? If not, please remove, otherwise be more specific in terms of the type of exception